src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -79,47 +89,44 @@
       <TestCopyLocal Include="@(_ExistingReplacementCandidate)" />
     </ItemGroup>
 
-    <ItemGroup Condition="'$(BUILDTOOLS_OVERRIDE_RUNTIME)' != ''">
-      <OverridenRuntimeContent Include="$(BUILDTOOLS_OVERRIDE_RUNTIME)/*.*" />
-      <TestCopyLocal Include="@(OverridenRuntimeContent)" />
-     </ItemGroup>
+    <!-- Remove files which are in SourcesToCopyToTestDir because these are copied to the test dir in CopyTestToTestDirectory. -->
+    <RemoveCommonItems SourceItems="@(TestCopyLocal)"
+                       DestinationFolder="$(TestPath)"
+                       ReferenceItems="@(SourcesToCopyToTestDir->'%(Destination)')">
+      <Output TaskParameter="UniqueItems" ItemName="_TestDependencies" />
+    </RemoveCommonItems>
```

Comment:
Is this task looking in DestinationFolder for a specific set of files (ReferenceItems), and if they are present, removing them from SourceItems?  Do you need a custom task for this?  Couldn't you just use msbuild for this?